test

```diff
@@ -8,6 +8,14 @@
 #
 # PKG=./wal ./test
 # PKG=snap ./test
+#
+# Environment variables of interest:
+#
+# E2E_TIMEOUT          - integration_pass e2e test timeout.
+# INTEGRATION_TIMEOUT  - integration_pass integration test timeout.
```

Comment:
are you using these? I don't think this will work on slower machines since there are a lot of tests that have their own timeout checks like `<-time.After(time.Second)`. The tests should just fail if the machine is too slow; etcd wouldn't work very well if the tests are too slow.